reference/fleet/example-kubernetes-fleet-managed-agent-helm.md

```diff
@@ -38,47 +58,54 @@ To get started, you need:
 3. In the **Add agent** UI, specify a policy name and select **Create policy**. Leave the **Collect system logs and metrics** option selected.
 4. Scroll down in the **Add agent** flyout to the **Install Elastic Agent on your host** section.
 5. Select the **Linux TAR** tab and copy the values for `url` and `enrollment-token`. You’ll use these when you run the `helm install` command.
-6. Open a terminal shell and change into a directory in your local clone of the `elastic-agent` repo.
-7. Copy this command.
+6. Open a terminal shell on your local system where the Helm tool is installed and you have access to the {{k8s}} cluster.
+7. Copy and prepare the command to install the chart:
 
     ```sh
-    helm install demo ./deploy/helm/elastic-agent \
+    helm install demo elastic/elastic-agent \
     --set agent.fleet.enabled=true \
-    --set agent.fleet.url=<Fleet-URL> \
-    --set agent.fleet.token=<Fleet-token> \
+    --set system.enabled=true \
+    --set agent.fleet.url=<Fleet-URL> \ # Substitute Fleet-URL with the URL that you copied earlier
+    --set agent.fleet.token=<Fleet-token> \ # Substitute Fleet-token with the enrollment token that you copied earlier.
     --set agent.fleet.preset=perNode
     ```
 
     Note that the command has these properties:
```

Comment:
if you want to leave this list here, we can drop `note that` 

```suggestion
    The command has these properties:
```

Reply:
updated locally, thanks!